third_party/optimized-svg-icons/amp-social-share-svgs.css

```diff
@@ -35,3 +35,7 @@
 }
+
+.amp-social-share-system {
+  background-image: url('data:image/svg+xml;charset=utf8,%3Csvg%20fill=%22#ffffff%22%20height=%2224%22%20viewBox=%220%200%2024%2024%22%20width=%2224%22%20xmlns=%22http://www.w3.org/2000/svg%22%3E%3Cpath%20d=%22M0%200h24v24H0z%22%20fill=%22none%22/%3E%3Cpath%20d=%22M18%2016.08c-.76%200-1.44.3-1.96.77L8.91%2012.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5%201.25.81%202.04.81%201.66%200%203-1.34%203-3s-1.34-3-3-3-3%201.34-3%203c0%20.24.04.47.09.7L8.04%209.81C7.5%209.31%206.79%209%206%209c-1.66%200-3%201.34-3%203s1.34%203%203%203c.79%200%201.5-.31%202.04-.81l7.12%204.16c-.05.21-.08.43-.08.65%200%201.61%201.31%202.92%202.92%202.92%201.61%200%202.92-1.31%202.92-2.92s-1.31-2.92-2.92-2.92z%22/%3E%3C/svg%3E');
```

Comment:
Done: all instances of `charset=utf8` in this file have been changed to `charset=utf-8`. (Also added a charset to `.amp-social-share-whatsapp`, which was missing it completely.)